python/PyQt6/core/auto_additions/qgis.py

```diff
@@ -11041,6 +11041,21 @@
 """
 # --
 Qgis.MouseHandlesAction.baseClass = Qgis
+# monkey patching scoped based enum
+Qgis.PointCloudZoomOutRenderBehavior.RenderExtents.__doc__ = "Render just point cloud extends when zoomed out"
+Qgis.PointCloudZoomOutRenderBehavior.RenderOverview.__doc__ = "Render overview point cloud when zoomed out"
+Qgis.PointCloudZoomOutRenderBehavior.RenderOverviewAndExtents.__doc__ = "Render extent polygons over overview point cloud"
+Qgis.PointCloudZoomOutRenderBehavior.__doc__ = """Point cloud zoom out options
+
+.. versionadded:: 3.42
+
+* ``RenderExtents``: Render just point cloud extends when zoomed out
```

Comment:
```suggestion
* ``RenderExtents``: Render only point cloud extents when zoomed out
```
I think "only" is what is usually used.
Also is "point cloud extents" different from "extent polygons" mentioned two lines below? If not, I'd suggest to use the same expression or clarify the difference. It would avoid this question from end users.

Reply:
Fixed!